tooling/bundler/src/bundle.rs

```diff
@@ -63,6 +63,19 @@ pub fn bundle_project(settings: Settings) -> crate::Result<Vec<Bundle>> {
     warn!("Cross-platform compilation is experimental and does not support all features. Please use a matching host system for full compatibility.");
   }
 
+  // Sign main windows binary before the bundling step in case neither wix and nsis bundles are enabled
+  #[cfg(target_os = "windows")]
+  {
+    let main_binary = settings
+      .binaries()
```

Comment:
should we take this opportunity to sign all binaries instead of just the main one?

Reply:
yeah, why not. gonna look into that asap

Reply:
2 Questions:
- Do all the binaries in settings.binaries need to be signed? I don't know what binaries it bundles other than the main app
- I think we need a setting to skip signing sidecars (or some of them) in case they were already signed beforehand. in case it was already signed by a third party or something idk

Reply:
Like how some projects implement the updater as a separate process. But I don't remember seeing anyone using it (except whoever requested this during our alpha, I think it was like a local server/client app).